AISKU/src/Init.ts

```diff
@@ -13,7 +13,7 @@ function _logWarn(aiName:string, message:string) {
     // TODO: Find better place to warn to console when SDK initialization fails
     var _console = typeof console !== Undefined ? console : null;
     if (_console && _console.warn) {
-        _console.warn('Failed to initialize AppInsights JS SDK for instance ' + aiName + message);
+        _console.warn('Failed to initialize AppInsights JS SDK for instance ' + (aiName || '<unknown>') + ' - ' + message);
```

Comment:
Just formatting this error a little better, which mostly occurs on when the environment doesn't contain "window" or "JSON" implementations.